src/Public/js/sis.messages.js

```diff
@@ -86,7 +86,14 @@ var Messages = function () {
 
                 locationTd.innerHTML = icon_location;
                 unreadTd.innerHTML = icon_unread;
-                titleTd.innerHTML = data[i]['title'];
+
+                var longTitle = data[i]['title'];
+                var trimmedTitle = longTitle.substring(0, 120);
+                if (longTitle != trimmedTitle) {
+                    trimmedTitle = trimmedTitle + '... <a>Read more</a>';
```

Comment:
If someone clicks on read more, does it still work, or does it's anchor tag with no destination override to do nothing?